### DISCUSSION TITLE
2024/04/10/%E6%9C%89%E6%9C%BA%E5%8F%8D%E5%BA%94%E6%B1%87%E6%80%BB-%E5%81%B6%E8%81%94%E5%8F%8D%E5%BA%94

### DISCUSSION THREAD

**Discussion by giscus** [WindskyMekdull/WindskyMekdull.github.io#8] [Announcements]
# 2024/04/10/%E6%9C%89%E6%9C%BA%E5%8F%8D%E5%BA%94%E6%B1%87%E6%80%BB-%E5%81%B6%E8%81%94%E5%8F%8D%E5%BA%94

前言偶联反应，又名耦合反应、偶合反应、耦联反应，是有机化学中的一类重要反应。广义的偶联反应是指将两个或以上的有机单位连为一体的反应，按此定义，酯化反应、醚化反应等都可以归为此类。而狭义的偶联反应指有机金属化合物参与或催化的 $C-C$ 或 $C-X$ 成键反应。可以看出，偶联反应的定义并不十分明确，且与缩合反应的...

https://mekdull.netlify.app/2024/04/10/%E6%9C%89%E6%9C%BA%E5%8F%8D%E5%BA%94%E6%B1%87%E6%80%BB-%E5%81%B6%E8%81%94%E5%8F%8D%E5%BA%94.html

**Comment by AlexEinstein** [comment]
hahahahahahahah
哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈

**Comment by AlexEinstein** [comment]
看不懂看不懂看不懂看不懂看不懂看不懂看不懂ing

**Comment by LYF9139** [comment]
金克拉得瑟啊微软天气我提前去问他屁哦很绝望贴哦一人去陪她喝酒去湖泊犹太人商业化阿尔忒为人类虎鱼为朋友也萨嘎哇如果i后退阿婆越爱朋友哈尔埃尔哈特怕欸普惠特瑞哈购房贷款了金额随楼市过热世界观和苏若国内各个奥恩圭饿啊日奴欸旁的立刻就三个年级阿尔爱国就让怕鞥啊劲儿啊人体图图就是快乐安居房给你机会科技人员和那天风很大啊饿哦和解放大卡这家公司独特如果大哥安居客火烈鸟贵哦热狗率年度爱人突然啊是对干哈就拿绿卡入股哦却赫然以哦徒惹哦亲退平台接入取货推荐给覅十多个会额外和体育健儿给大家看认为躯体爱国对饿啊让看两个人就开始德国坦克就是二四六他会数据库的泛滥格瑞通过了一节课属地化监管空白你就不能是党和国家半年后是那个本节课图书价格多少看过呢过呢是
